org.eclipse.dawnsci.analysis.examples/src/org/eclipse/dawnsci/analysis/examples/pipelines/PipelineExamples.java

```diff
@@ -34,7 +34,7 @@
 
 	private static IOperationService service;
 	
-	public static void setOperationService(IOperationService s) {
+	public void setOperationService(IOperationService s) {
 		service = s;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)